package.json

```diff
@@ -220,6 +220,7 @@
     "test:client:integration:cognito-local:ci": "npm run test:client:_integration -- --coverageDirectory=./coverage-integration-${CIRCLE_NODE_INDEX} ${TESTFILES}",
     "test:client:integration:ci": "npm run test:client:_integration -- --coverageDirectory=./coverage-integration-${CIRCLE_NODE_INDEX} ${TESTFILES}",
     "test:client:unit": "jest --config web-client/jest-unit.config.ts --testMatch '**/web-client/src/**/?(*.)+(spec|test).[jt]s?(x)'",
+    "test:client:unit:file": "jest --config web-client/jest-unit.config.ts --testMatch '**/web-client/src/**/?(*.)+(spec|test).[jt]s?(x)' --coverage=false",
```

Comment:
thank you for these.

Reply:
Hmm, it appears to work on the file that I specify. Perhaps chat about this more online?